apps/api/agents/setup-venv.sh

```
@@ -0,0 +1,124 @@
+if [ -f "$REQUIREMENTS_FILE" ]; then
+    echo "📋 Installing Python dependencies from $REQUIREMENTS_FILE..."
+    pip install -r $REQUIREMENTS_FILE
+else
+    echo "❌ Requirements file not found: $REQUIREMENTS_FILE"
+    exit 1
+fi
+
+# Install additional healthcare-specific dependencies
+echo "🏥 Installing healthcare-specific dependencies..."
+pip install --upgrade \
+    python-jose[cryptography] \
```

Shellcheck (0.11.0)

[info] Ranges can only match single chars (mentioned due to duplicates).

(SC2102)